samples/13.b.customization-smart-display/README.md

```diff
@@ -0,0 +1,248 @@
+
+# Things to try out
+
+-  Click the microphone button and say "what's the weather"
+
+# Overview
+
+This sample uses the base components of Web Chat to build a new user interface that suit the voice assistant scenario. In this case, we are creating a smart display which shows:
+
+1. Time of the day and weather forecast
+1. Microphone button to initiate speech-to-text
+1. Interims while the user is speaking
+1. Last message from the bot, including Adaptive Cards
+
+Notice that traditional Web Chat "pieces" are missing: the transcript and send box are not present on this app.
+
+The intent of this sample is to show how you can pick and choose what components you want to use, allowing for expansion and modification of how traditional Web Chat works. If you have a significantly different design spec for your bot, but would still like to use Web Chat, looking at this sample is a great place to start.
+
+To see what components are available for customization, please take a look at the [components directory](https://github.com/microsoft/BotFramework-WebChat/tree/master/packages/component/src) of the Web Chat repo. You are welcome to import any one of these into your app and make modifications, or build an entirely new component from scratch.
+
+First, let's take a look at the `App.js` file.
+
+Note that this is the component that builds up the other components we will create. We will need to import `<Composer>` from Web Chat to form the container of our application. Inside the composer, our imported components that we will create will be utilized within that container.
+
+## `App.js`
+
+[`App.js`](https://github.com/microsoft/BotFramework-WebChat/blob/master/samples/13.b.customization-smart-display/src/App.js) will set up the container for hosting Web Chat components. The container will be using Direct Line Speech channel.
+
+`useEffect` and `useState` hooks are used to create the Direct Line Speech adapter set asynchronously. After the adapter set are created, they will be used to create the container.
+
+```jsx
+const [directLineSpeechAdapters, setDirectLineSpeechAdapters] = useState();
+
+useEffect(() => {
+   (async () =>
+   setDirectLineSpeechAdapters(
+      await createDirectLineSpeechAdapters({
+         fetchCredentials: fetchCognitiveServicesCredentials
+      })
+   ))();
+}, [setDirectLineSpeechAdapters]);
+
+return (
+   !!directLineSpeechAdapters && (
+   <Composer {...directLineSpeechAdapters}>
+      <SmartDisplay />
+   </Composer>
+   )
+);
+```
+
+## `SmartDisplay.js`
+
+[`SmartDisplay.js`](https://github.com/microsoft/BotFramework-WebChat/blob/master/samples/13.b.customization-smart-display/src/SmartDisplay.js) will set up all the components for smart display, including clock, speech interims, bot response, and microphone button. The component will also keep track of the last read activity. When the user click on the microphone button, it will mark the last bot activity as read.
```

Comment:
```suggestion
[`SmartDisplay.js`](https://github.com/microsoft/BotFramework-WebChat/blob/master/samples/13.b.customization-smart-display/src/SmartDisplay.js) will set up all the components for smart display, including clock, speech interims, bot response, and microphone button. The component will also keep track of the last read activity. When the user clicks on the microphone button, it will mark the last bot activity as read.
```